make verify FAILED

### Which jobs are failing?

root@LAPTOP-6IH0BQSV:/home/jiacheng/kubernetes# hack/make-rules/../../hack/verify-conformance-requirements.sh
go version go1.20 linux/amd64
hack/conformance/check_conformance_test_requirements.go:29:2: cannot find package "github.com/spf13/cobra" in any of:
        /home/jiacheng/kubernetes/_output/local/.gimme/versions/go1.20.linux.amd64/src/github.com/spf13/cobra (from $GOROOT)
        /root/go/src/github.com/spf13/cobra (from $GOPATH)
Errors from lint:


Please review the above warnings.

### Which tests are failing?

verify-conformance-requirements.sh

### Since when has it been failing?

02-18-2023

### Testgrid link

_No response_

### Reason for failure (if possible)

_No response_

### Anything else we need to know?

_No response_

### Relevant SIG(s)

/sig

--- a/hack/verify-conformance-requirements.sh
+++ b/hack/verify-conformance-requirements.sh
@@ -32,7 +32,7 @@ cd "${KUBE_ROOT}"
 
 errors=()
 # Check conformance tests follow the requirements as https://git.k8s.io/community/contributors/devel/sig-architecture/conformance-tests.md#conformance-test-requirements
-if ! failedLint=$(go run "${KUBE_ROOT}"/hack/conformance/check_conformance_test_requirements.go "${KUBE_ROOT}"/test/e2e/)
+if ! failedLint=$(GO111MODULE=on GOPROXY=off go run hack/conformance/check_conformance_test_requirements.go "${KUBE_ROOT}"/test/e2e/)
 then
   errors+=( "${failedLint}" )
 fi

